README.md

```diff
@@ -110,6 +115,64 @@ playlist. Input playlists are generally referred to by their URI, or
 unique identifier, which can be found by right-clicking on a playlist
 and going to `Share > Copy Spotify URI`.
 
+## Graphical Editor
+
+Power Playlists includes a modern web-based graphical editor that makes creating and editing playlist configurations much easier than manually writing YAML files. The editor provides a visual, node-based interface with drag-and-drop functionality.
+
+### Launching the Editor
+
+**Basic usage** - Automatically discover and load configurations:
+```shell
+> power-playlists edit
+```
+
+**Load specific configuration**:
+```shell
+> power-playlists edit --userconf ~/.power-playlists/userconf/myplaylist.yaml
+```
+
+**Use different app configuration**:
+```shell
+> power-playlists edit --appconf ~/my-app-config.yaml
+```
+
+### Features
+
+The graphical editor provides:
+
+- **Modern web interface** - Opens automatically in your default browser with a clean, professional design
+- **Visual node editor** - Color-coded nodes with intuitive icons (♫ for playlists, ❤️ for filters, 📤 for outputs)
+- **Drag & drop** - Easily reposition nodes to organize your playlist flow
+- **Curved connection lines** - Visual connections show dependencies between nodes
+- **Smart property editing** - Structured forms with dropdowns, checkboxes, and validation
+- **Real-time validation** - Prevents saving invalid configurations with detailed error messages
+- **Automatic discovery** - Finds existing configurations and provides a selector when multiple exist
+
+### Using the Editor
+
+1. **Adding nodes**: Click "Add Node" and select the type of node you want (playlist, filter, combiner, etc.)
+2. **Editing properties**: Double-click any node to edit its properties using structured forms
+3. **Connecting nodes**: Nodes are automatically connected based on their input/output relationships
+4. **Saving**: Use "Save" to update the current file or "Save As" to create a new configuration
+
+### Node Types Available
+
+The editor supports all Power Playlists node types:
```

Comment:
I believe we don't currently support dynamic template nodes in the graphical editor. Call out this caveat here.